geopandas/vectorized.pyx

```diff
@@ -936,12 +781,12 @@ cpdef binary_geo(str op,
                 else:
                     out[idx] = 0
 
-    return GeometryArray(out)
+    return out
 
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-cdef buffer(np.ndarray[np.uintp_t, ndim=1, cast=True] geoms, double distance,
+cpdef buffer(np.ndarray[np.uintp_t, ndim=1, cast=True] geoms, double distance,
```

Comment:
@mrocklin I changed a few `cdef` to `cpdef` to use them in `array.pyx`. 
I assume I can avoid this by creating a `vectorized.pxd` with their declarations, so I can `cimport vectorized` in `array.pyx`, but since they are there only used in class methods (of GeometryArray) and not in for loops, I think this will not really make a difference?